### ISSUE TITLE
Package on conda-forge

### ISSUE THREAD

**Issue by fjetter** [dask-contrib/dask-deltatable#26]
It would be nice to have this package on conda-forge as well

**Comment by jrbourbeau** [comment]
Yeah, agreed. Looking at the dependencies here, `deltatable` not being on `conda-forge` looks to be the only blocker. There's an open PR for adding it over in `staged-recipes` which I've pinged https://github.com/conda-forge/staged-recipes/pull/22162#pullrequestreview-1481813820. Let's see if there's a response. If not, we can open up our own PR. My current sense is this is definitely something we want, but can also wait a few days (feel free to correct me if that's not the case)

**Comment by charlesbluca** [comment]
Opened https://github.com/conda-forge/staged-recipes/pull/23386 to add `deltalake` to conda-forge; once that's in, happy to help with any conda packaging needs for this package 🙂

**Comment by charlesbluca** [comment]
`deltalake` PR is in and packages are now published:

https://anaconda.org/conda-forge/deltalake/files

I've gone ahead and opened https://github.com/conda-forge/staged-recipes/pull/23636 to add `dask-deltatable` now that we're unblocked - feel free to take a look in case there's anything I'm missing there

**Comment by charlesbluca** [comment]
The above PR is in and packages are now published 🎉 

https://anaconda.org/conda-forge/dask-deltatable/files

Anything else that needs to happen here or should we be good to close this?

**Comment by jrbourbeau** [comment]
Thanks for handling that @charlesbluca! 

It might be good to update the README with conda install instructions (right now we just have pip). But after that, I think we should be good to close
